etna/transforms/timestamp/holiday.py

```diff
@@ -27,8 +71,9 @@ class HolidayTransform(IrreversibleTransform):
     """
     HolidayTransform generates series that indicates holidays in given dataset.
 
-    In ``binary`` mode shows the presence of holiday in that day. In ``category`` mode shows the name of the holiday
-    with value "NO_HOLIDAY" reserved for days without holidays.
+    - In ``binary`` mode shows the presence of holiday in that day.
+    - In ``category`` mode shows the name of the holiday with value "NO_HOLIDAY" reserved for days without holidays.
+    - In the ``days_count mode``, calculate frequency of holidays if frequency is weekly, monthly, quarterly or yearly .
```

Comment:
Here we should clarify how exactly do we work with each frequency. It can't be understood now without looking at the source code.